keras/src/ops/operation.py

```diff
@@ -97,6 +100,7 @@ def __new__(cls, *args, **kwargs):
         to manually implement `get_config()`.
         """
         instance = super(Operation, cls).__new__(cls)
+        vars(instance)['_object__state'] = nnx.object.ObjectState()
```

Comment:
Oh, this cannot go here. This class is cross-backend and `Operations` are required to be stateless. This is probably what is making all tests fail.

Move to `JaxLayer`.